thirdparty/openapi3/schemas/ThirdpartyAuthorizationIDPostResponse.yaml

```diff
@@ -0,0 +1,34 @@
+title: ThirdpartyAuthorizationIDPostResponse
+type: object
+description: The response for POST /thirdpartyRequests/transactions/{id}/authorizations request.
+properties:
+  challenge:
+    type: string
+    description: The original Challenge Object as a JSON string
+  value:
+    allOf:
+      - $ref: '../../../v1.0/openapi3/schemas/BinaryString.yaml'
+    description: 'Base64 encoded binary string - the signed challenge'
+  consentId:
+    allOf:
+      - $ref: '../../../v1.0/openapi3/schemas/CorrelationId.yaml'
+    description: >
+      Common ID between the PISP and FSP for the Consent object
+      This tells DFSP and auth-service which constent allows the PISP to initiate transaction.
+  sourceAccountId:
+    allOf:
+      - $ref: './AccountAddress.yaml'
+    description: DFSP specific account identifiers, e.g. `dfspa.alice.1234`
+  status:
+    type: string
+    enum:
+      - VERIFIED
+    description: The status of the authorization.
+  currentState:
```

Comment:
currentState is only on Outbond. @sridharvoruganti could you point out Kevin to proper core interface?